ChiTech/ChiPhysics/FieldFunction/fieldfunction_exportvtk.cc

```diff
@@ -118,13 +119,19 @@ void chi_physics::FieldFunction::WritePVTU(const std::string& base_filename,
 
   bool is_global_mesh =
     chi_mesh::GetCurrentHandler()->volume_mesher->options.mesh_global;
+  
+  // Cut off path to base_filename
+  std::string filename_short;
+  std::istringstream iss(base_filename);
+  while (std::getline(iss, filename_short, '/'))
```

Comment:
are you trying to do this? If so, what's wrong with my one-liner?

std::string filename_short = base_filename.substr(base_filename.find_last_of("/\\") + 1)

I let @Naktakala  decide what he prefers

Reply:
The one liner needs to escape the "\\" sybol. I just godbolted both versions:

### PR Version:
```c++
std::string song = "Maya_hi/Maya_hu/Maya_ho/Maya_haha/dragos_din_tei.old";

std::string filename_short;
std::istringstream iss(song);
while (std::getline(iss, filename_short, '/'))

std::cout << filename_short;
```
```
>>Maya_hiMaya_huMaya_hoMaya_hahadragos_din_tei.old
```

### Ragusa version
```c++
std::string song = "Maya_hi/Maya_hu/Maya_ho/Maya_haha/dragos_din_tei.old";

std::string filename_short = song.substr(song.find_last_of("/\\")+1);

std::cout << filename_short;
```

```
>>dragos_din_tei.old
```

So please change to second version.

Reply:
@Naktakala I am still useful !!

Reply:
@Naktakala, you get a bunch of junk in the PR version because you left out the continue, allowing the cout to execute with each iteration. After the loop, the variable filename_short holds the desired value. I agree that the one-liner is more concise, however.

Reply:
@pbehne My bad.